Sources/Purchasing/Configuration.swift

```diff
@@ -207,19 +212,32 @@ import Foundation
 
 // MARK: - Public Keys
 
-internal extension Configuration {
+public extension Configuration {
 
-    /// Defines how strict ``EntitlementInfo`` verification ought to be.
-    @objc(RCEntitlementVerificationLevel)
-    enum EntitlementVerificationLevel: Int {
+    // TODO: link to docs
 
-        /// The SDK will perform no entitlement verification.
+    /// Defines how strict ``EntitlementInfo`` validation ought to be.
+    ///
+    /// ### Related Symbols
+    /// - ``EntitlementValidation``
+    /// - ``Configuration/Builder/with(entitlementValidationMode:)``
+    /// - ``EntitlementInfos/validation``
+    @objc(RCEntitlementValidationMode)
+    enum EntitlementValidationMode: Int {
+
+        /// The SDK will not perform any entitlement validation.
         case disabled = 0
 
-        /// The SDK will verify entitlements, but will not fail to parse them if verification failed.
+        /// Enable entitlement validation.
+        ///
+        /// If validation fails, this will be indicated with ``EntitlementValidation/failedValidation``
+        /// but parsing will not fail.
```

Comment:
Maybe clarify that entitlements will be granted? And maybe some description of the use cases. SOmething like:

```suggestion
        /// Enable entitlement validation but continue granting entitlements on failure.
        /// This can be useful if you want to customize your reaction to validation failures but still grant access.
        ///
        /// If validation fails, this will be indicated with ``EntitlementValidation/failedValidation``
        /// but parsing will not fail.
```

Reply:
Good idea, added.